src/app.ts

```diff
@@ -146,4 +189,34 @@ export default (
       method: 'deleteSecret',
     });
   });
-};
+
+  app.on("issues.opened", async(context: any) => {
+    var issueComment = context.issue({
+           body: "Thanks for submitting onboarding request!",
+         });
+    
+    var data;
+    try {
+      data = await parse(context);
+    } catch {
+      app.log.info('Issue was not created using Issue form template (the YAML ones)');
+      issueComment = context.issue({body: "Sorry, there was an error submitting your request."})
+      return context.octokit.issues.createComment(issueComment);
+    }
+
+    const body:string = context.payload.issue["body"];
+    if (body.includes("### Target cluster")){
```

Comment:
What is this `if` check used for?

Reply:
To check if the template used was the one for onboarding.